src/app/views/dashboard.layout.html

```diff
@@ -76,6 +76,12 @@
     <li class="sidebar-list" ng-if="::main.isDashboardTemplatesEnabled" ui-sref-active="active">
       <a ui-sref="dashboard.dashboard-templates">{{'mnoe_admin_panel.dashboard.layout.sidebar.dashboard_templates' | translate}}<span class="menu-icon fa fa-bar-chart"></span></a>
     </li>
+    <div ng-if="::main.adminConfig.areSettingsEnabled()" ng-show="main.user.admin_role == 'staff'">
```

Comment:
Duplicate code.
Refactor the menu below to only display the "settings-apps" submenu when `role == 'staff'`